ibis/sql/mssql/api.py

```diff
@@ -0,0 +1,125 @@
+from ibis.sql.alchemy import to_sqlalchemy
+from ibis.sql.mssql.client import MSSQLClient
+from ibis.sql.mssql.compiler import (  # noqa: F401, E501
+    compiles,
+    dialect,
+    rewrites,
+)
+
+
+def compile(expr, params=None):
+    """Compile an ibis expression to the MSSQL target.
+
+    Parameters
+    ----------
+    expr : ibis.expr.types.Expr
+        The ibis expression to compile
+    params : dict or None
```

Comment:
I think here you can use `params : dict, optional`

Reply:
Fixed.